docs/assets/main.css

```
@@ -1105,182 +1181,263 @@
+    background-color: color-mix(in srgb, #fff 30%, transparent);
+    @supports (color: color-mix(in lab, red, red)) {
+      background-color: color-mix(in oklab, var(--color-white) 30%, transparent);
+    }
   }
   .bg-linear-to-br {
-    --tw-gradient-position: to bottom right in oklab,;
+    --tw-gradient-position: to bottom right;
+    @supports (background-image: linear-gradient(in lab, red, red)) {
+      --tw-gradient-position: to bottom right in oklab;
+    }
     background-image: linear-gradient(var(--tw-gradient-stops));
```

Biome (1.9.4)

[error] Unexpected nonstandard direction

You should fix the direction value to follow the syntax.
See MDN web docs for more details.

(lint/correctness/noInvalidDirectionInLinearGradient)